README.md

```diff
@@ -10,15 +10,13 @@ This is the code which runs the DevCongress Jobs website, which lives at [jobs.d
 
 Install Docker and Docker Compose (Docker Compose comes with Docker on Windows and MacOS)
 
-### Start Project
+### Run Project
 
-Launch docker with `docker-compose up`
+Run `./scripts/create.sh` if on linux.
 
-## Run db:seed
-
-```sh
-docker-compose run --rm web rake db:seed
-```
+Otherwise,
+- run `docker-compose up --build -d` to start the containers
```

Comment:
Its worth explaining what `-d` does, because not everyone will want the container to be running in the background. Maybe also add how to stop that container running in the background if you wanted to.

Reply:
Sure.